pkgs/development/libraries/clipp/clipp.pc

```diff
@@ -0,0 +1,9 @@
+prefix=@out@
```

Comment:
Did you upstream this file?

Reply:
I'm offering it upstream (https://github.com/muellan/clipp/pull/33), but it's not accepted yet. If the next release provides pkg-config support I'll update and remove it from nixpkgs.